how_to_build.md

```diff
@@ -35,6 +35,8 @@ At the time of writing the build is only confirmed to work on Windows using the
 
 1. Install gems:
 
+    Using a command prompt, navigate to your clone root folder and execute:

```

Comment:
It's a nitpick, but I don't like the repitition of "using a command prompt" here. It's already two items above, and if we have it here, we should also have it in the item directly above. (We also repeat it in the next section.) We can do a more general clean up later, but for now I'd reduce this to something like:

> 1. Install gems (after navigating to your clone root folder):